Tools/Cli-LoRa-Device-Provisioning/Helpers/IoTDeviceHelper.cs

```diff
@@ -1353,5 +1332,29 @@ public async Task<bool> RemoveDevice(string devEui, ConfigurationHelper configur
             Console.WriteLine("done.");
             return true;
         }
+
+        private static async Task<bool> ExecuteWithIotHubErrorHandlingAsync(Func<Task> executeAsync, string errorMessageContext = null)
```

Comment:
[nit] do you think this makes sense to keep as a separate method, given that this is referenced only once?

Reply:
Done in 77aeb36e4dc075aec50b2f18c5c7b254f086a76f